apps/emqx_management/src/emqx_mgmt_api_configs.erl

```diff
@@ -38,12 +38,21 @@
 -define(OPTS, #{rawconf_with_defaults => true, override_to => cluster}).
 -define(TAGS, ["Configs"]).
 
+-if(?EMQX_RELEASE_EDITION == ee).
+-define(ROOT_KEYS_EE, [
+    <<"file_transfer">>
```

Comment:
File Transfer has its [own configuration update mechanism](https://github.com/emqx/emqx/blob/35a313666095cd1fb8617a2e0108aed2b8820420/apps/emqx_ft/src/emqx_ft_api.erl#L120-L131) nowadays, will this change provide essentially the same thing? Or is this exactly the goal here?

Reply:
Thank you for the tip. Will check with frontend team to see if it's good to take swagger spec for reference.

Reply:
The `hotconf` schema json is used by the dashboard to render UI.